web-client/src/components/ClusterAccessRadio.tsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react'
+import {ClusterAccess} from "./types";
 
-export default function ClusterAccessRadio({
-  clusterAccess,
-  setClusterAccess
-}) {
+interface ClusterAccessRadioParameters {
+  readonly clusterAccess: string,
```

Comment:
Shouldn't `clusterAccess` be of type `ClusterAccess` instead of `string`?

Reply:
true, this one slipped away 👀